src/excmds.ts

```diff
@@ -2567,6 +2567,32 @@ export async function viewcontainers() {
             .replace(/ /g, "%20")
 }
 
+/** Opens the current tab outside the container
+
```

Comment:
Please add a link to https://github.com/mozilla/multi-account-containers/wiki/Moving-between-containers and, because no-one will read it, a precis that is short enough to appear in the interactive help warning people that it might be a bad idea.